.github/build_real.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 set -e
-pip install .[testing]
+pip install -e .[testing]
```

Comment:
I guess my question for [this PR](https://github.com/mdolab/multipoint/pull/16) comes from here: why is this necessary / best practice?

Reply:
I have been redirected to the instructions, and I see how this makes sense... but at the same time it looks like it is not necessary for the IDWarp PR, I am getting a bit lost

Reply:
Interesting, perhaps this is not needed..